test/unit/dealer/ftx-dealer-wallet.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { redis } from "src/redis"
+import { FtxDealerWallet } from "src/dealer/FtxDealerWallet"
+
+beforeAll(async () => {
+  // avoids to use --forceExit and the need of a running redis
+  redis.disconnect()
```

Comment:
If we don't need redis for this file then we should rather have a way _not_ to start in except for the files were we need it instead of always starting it and then disabling. OR? Its better to explicitly require something (like a running redis) than implicitly require everything and then explicitly have to remove it when its not needed.

Reply:
This is part of what we must try to gradually migrate. The `src/dealer/FtxDealerWallet` does not use redis explicitly but imports other files that use it and there is no way (with the current structure) to handle "disconnect" (similar to the PR related to express httpServer).